web/src/less/create-shortcut-dialog.less

```diff
@@ -2,7 +2,7 @@
   @apply px-4;
 
   > .dialog-container {
-    @apply w-128 max-w-full;
+    @apply w-160 max-w-full;
```

Comment:
Please take a look into the mobile display.

Reply:
I changed it back to `w-128`. 

I checked the mobile style and found another bug, I'm going to open another PR to deal with it.

![image](https://user-images.githubusercontent.com/31177490/201519046-5b1b335c-6c27-4852-b9b1-878a37f5c1d0.png)
![image](https://user-images.githubusercontent.com/31177490/201519057-e13d2ed6-4783-447d-9632-c8adece4e10f.png)